src/JsonSchema/Model.elm

```diff
@@ -23,6 +24,7 @@ type alias BaseSchema extras =
     { extras
         | title : Maybe String
         , description : Maybe String
+        , examples : Maybe (List Encode.Value)
```

Comment:
Can we make this a simple list instead of a list wrapped in a maybe? The list being empty should be enough indication that there are no examples.